### PR TITLE
main/libressl: fix curl error 77

### PR DESCRIPTION
**Pull request by itoffshore**
fixes:
```
	curl: (77) error setting certificate verify locations:
	  CAfile: /etc/ssl/certs/ca-certificates.crt
	  CApath: none
```

### CODE REVIEW

**Comment by jirutka** [comment]
@ncopa or @Shizmob, could you please review this PR?

File: main/libressl/APKBUILD [hunk 1]
```diff
@@ -29,19 +29,21 @@ build() {
...
 }
 
 package() {
 	cd "$builddir"
-	make DESTDIR="$pkgdir" install || return 1
+	make DESTDIR="$pkgdir" install
 	install -Dm644 COPYING "$pkgdir"/usr/share/licenses/$pkgname/COPYING
+	# fix curl: (77) error setting certificate verify locations:
+	mkdir -p "$pkgdir"/etc/ssl/certs
+	ln -s /etc/ssl/cert.pem "$pkgdir"/etc/ssl/certs/ca-certificates.crt
```
Comment (Shizmob) [inline main/libressl/APKBUILD]: I think a relative symlink would be better, like `ln -s ../cert.pem "$pkgdir"/etc/ssl/certs/ca-certificates.crt`.

File: main/libressl/APKBUILD [hunk 2]
```diff
@@ -57,4 +59,9 @@ _libs() {
 	done
 }
 
+check() {
+	cd "$builddir"
+	make check
```
Comment (Shizmob) [inline main/libressl/APKBUILD]: 👌

File: main/libressl/APKBUILD (hunk 1 above)
Comment (Shizmob) [inline main/libressl/APKBUILD]: Also, the comment could be generalised to "fix for programs that expect the Debian default location".

**Comment by itoffshore** [comment]
fixed & rebased

File: main/libressl/APKBUILD [hunk 3]
```diff
@@ -2,9 +2,9 @@
 # Contributor: Orion <systmkor@gmail.com>
 # Maintainer: Orion <systmkor@gmail.com>
 pkgname=libressl
-pkgver=2.5.3
+pkgver=2.5.4
 _namever=${pkgname}${pkgver%.*}
-pkgrel=0
+pkgrel=1
```
Comment (Shizmob) [inline main/libressl/APKBUILD]: If you're bumping the version, the pkgrel should be reset to 0 -- but that should be a separate commit if you do, possibly a separate PR, since I recall there being at least one problematic change in LibreSSL 2.5.4 w.r.t. `getrandom()` usage. I'd recommend leaving it at 2.5.3 for this PR.

**Comment by Shizmob** [comment]
Thanks! Missed one thing (see comment above), but the rest LGTM.

File: main/libressl/APKBUILD (hunk 3 above)
Comment (itoffshore) [inline main/libressl/APKBUILD]: had to change it `2.5.4` for it to merge on github

Comment (Shizmob) [inline main/libressl/APKBUILD]: Ah, it was already updated and GitHub was weird about showing changes -- nevermind then!

**Comment by clandmeter** [comment]
Am I missing something here? doesn't ca-certificates provide /etc/ssl/certs/ca-certificates.crt? And curl depends on ca-certificates.

**Comment by Shizmob** [comment]
Yes, it should as part of the post-install trigger, come to think of it... that's weird.

**Comment by itoffshore** [comment]
@clandmeter - `ca-certificates` [does not exist in Alpine](https://pkgs.alpinelinux.org/contents?file=ca-certificates.crt&path=&name=ca-certificates&branch=edge&repo=main&arch=x86_64).

**Comment by clandmeter** [comment]
It is via post install.
How did you add curl? If this file does not exist on your system the post install for CA certificate failed somehow.

**Comment by itoffshore** [comment]
@clandmeter - ok this seems to be fixed now after checking my `ssl`
